GuyHoozdis/hypermodern-python
# Chapter 1: Project Setup

# Why Are We Doing This

I came across the series of articles, [The Hypermodern Python Project][chapter-1], and wanted to glean some of the difference that he was doing compared to the Python Packaging Authority with their [Sample Project][sample-project] repo.

Article: [Hypermodern Python Chapter 1: Project Setup][chapter-1]

[chapter-1]: https://cjolowicz.github.io/posts/hypermodern-python-01-setup/
[sample-project]: https://github.com/pypa/sampleproject

# Test Plan

1. Install the package: `poetry install`
1. Verify `poetry run hypermodern-python --help` displays the CLI help.
1. Verify `poetry run hypermodern-python --version` displays the application version (i.e. `0.1.0` in this commit)
1. Verify `poetry run hypermodern-python` writes out the title of a random wiki article, in green, followed by a summary of the article.
